### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by roguemacaroon**
updates to headings
resolved merge conflicts from development

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -150,7 +150,7 @@
 			<!-- image of girl with dandelion -->
 			<!-- this image is only displayed on mobile and does not use a source set like the other sections -->
 			<img
-				src="./assets/images/pages/home/mobile/hero-home-mobile-2.jpg"
+				src="./assets/images/pages/home/mobile/hero-home-mobile-compressed.jpg"
```
Comment (roguemacaroon) [inline index.html:153]: changed the image to a more compressed version

File: index.html [hunk 2]
```diff
@@ -202,7 +212,7 @@ <h1 class="h3-styles">Preserving Tomorrow by Building Today</h1>
 				</picture>
 				<!-- "partnerships-paragraph" contains the header, paragraph text, and button on amaranth background-->
 				<div class="partnerships-paragraph">
-					<h4>Community Partnerships</h4>
+					<h2 class="h4-styles">Community Partnerships</h2>
```
Comment (roguemacaroon) [inline index.html:215]: changed h4 to h2 with a class of h4

**Comment by roguemacaroon** [comment]
noticed accessibility issues for wave, changes from home page were not added into development (the h tag changes)

**Review by maxharrisnet — APPROVED** [review]
Nice fixes, thanks!

**Review by SolivanLau — APPROVED** [review]
Went through with Trish! Looks good. thanks for Accessibility Fixes!

**Review by gurjeetsingh — APPROVED** [review]
looks great!

**Pull request merged**